OsmAnd/res/values/strings.xml

```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Try Wikivoyage</string>
```

Comment:
The string is used for file name, not for a button

Reply:
As in "Wikivoyage sample"? As in "A sample of Wikivoyage". I think I am missing something.
What does the file do?

Reply:
It should be "Wikivoyage example"